pkg/webhook/nginxingress.go

```diff
@@ -177,12 +193,13 @@ func (n *nginxIngressResourceValidator) Handle(ctx context.Context, req admissio
 
 	lgr.Info("decoding NginxIngressController resource")
 	var nginxIngressController approutingv1alpha1.NginxIngressController
-	if err := n.decoder.Decode(req, &nginxIngressController); err != nil {
+	if err = n.decoder.Decode(req, &nginxIngressController); err != nil {
```

Comment:
not related to this pr but could you add a log statement in this block? noticed I missed one

Reply:
added